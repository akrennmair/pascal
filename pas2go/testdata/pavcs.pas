program test;

var pavcs : packed array [1..10] of 'g'..'p';
	i : integer;

begin
	for i := 1 to 10 do
		pavcs[i] := chr(i+ord('f'))
end.
