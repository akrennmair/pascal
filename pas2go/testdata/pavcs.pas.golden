package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	var (
		pavcs [10]int
		i     int
	)
	_ = pavcs
	_ = i

	for i = 1; i <= 10; i++ {
		pavcs[i-(1)] = system.Chr(i + int('f'))
	}
}
